site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -304,6 +304,58 @@ a ClusterQueue can borrow up to the sum of nominal quotas from all the
 ClusterQueues in the cohort. So for the yamls listed above, `team-b-cq` can 
 borrow `12+9` CPUs.
 
+### LendingLimit
```

Comment:
Can you make the website update another PR? Then this one is more dedicated about the code change.

Reply:
ok